remote_artifact_runtimes/03_spark.ipynb

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.                                image='docker-registry.default-tenant.app.jmglmvqnganv.iguazio-cd1.com/mlrun/func-test-notebooks-dani-spark-func:latest', 
</pre><p>docker-registry.default-tenant.app.jmglmvqnganv.iguazio-cd1.com is system specific, change this</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/test-notebooks/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>